metals/src/main/scala/scala/meta/internal/metals/doctor/ScalaProblem.scala

```diff
@@ -82,6 +88,11 @@ case object DeprecatedSbtVersion extends ScalaProblem {
     "Code navigation might not be supported in the future for this sbt version, " +
       s"please upgrade to at least ${BuildInfo.minimumSupportedSbtVersion} and reimport the build"
 }
+case object DeprecatedRemovedSbtVersion extends ScalaProblem {
+  override def message: String =
+    "Support for Scala is no longer being bugfixed for this sbt version, " +
+      s"please upgrade to at least ${BuildInfo.minimumSupportedSbtVersion} and reimport the build"
+}
```

Comment:
Again, I find `Support for Scala is no longer being bugfixed for this sbt version,` misleading. What do you think about `sbt version $sbtVersion uses Scala $scalaVersion which has stopped being supported by Metals. In order to get the best IDE support please upgrade...`. 

best IDE support/working IDE support - this part might need @ckipp01 wording skills ;)

Reply:
> sbt version $sbtVersion uses Scala $scalaVersion, which is no longer supported by Metals. To get the best support possible it's recommended to update.